src/lib/djerba/extract/r_script_wrapper.py

```diff
@@ -255,6 +256,16 @@ def preprocess_gep(self, gep_path):
                 writer.writerow(row)
         return out_path
 
+    def tab_file_is_empty(self, mavis_path):
+        """
+        Checks if .tab file provided is empty.
+        Returns False is not empty, returns True if empty.
```

Comment:
This doesn't actually return anything, instead it sets an instance variable. Which is fine, but I'd edit this comment and rename the method to something like `set_tab_empty_status`.